src/database/models/event.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
+
+from sqlalchemy import Column, String, TIMESTAMP, DateTime
+from .base import BaseModel
+
+
+class Event(BaseModel):
+    __tablename__ = "Event"
+
+    title = Column(String, nullable=False)
+    description = Column(String, nullable=True)
+    link = Column(String, nullable=True)
+    photo_id = Column(String, nullable=True)
+    status = Column(String, nullable=True)
+    date = Column(DateTime, nullable=False)
+    to_created = Column(TIMESTAMP, nullable=False)
+
+    def __init__(self, title, description, link, date):
```

Comment:
Should we add types like? 

```def __init__(self,title:str,description:str, link:str ,date: datetime.datetime.date)  -> None``` 
In case of date we should definetely show what type and consider add format of date 
I guess it is good practice.